src/libtls/tls_peer.c

```diff
@@ -1195,14 +1199,14 @@ static status_t send_client_hello(private_tls_peer_t *this,
 								  bio_writer_t *writer)
 {
 	tls_cipher_suite_t *suites;
-	bio_writer_t *extensions, *curves = NULL, *versions, *key_share, *signatures;
+	bio_writer_t *extensions, *curves = NULL, *versions, *signatures;
+	bio_writer_t *key_share;
```

Comment:
As mentioned, no need for this change (that single character the line is too long is fine :).